semeio/workflows/localisation/list_all_obs_and_model_param.py

```diff
@@ -0,0 +1,90 @@
+from res.enkf import ErtScript
+from res.enkf.enums.enkf_var_type_enum import EnkfVarType
+from semeio.workflows.localisation.local_script_lib import (
+    get_param_from_ert,
+    print_params,
+    check_if_ref_point_in_grid,
+)
+
+# import xtgeo
+
+debug_level = 1
+
+
+def debug_print(text):
+    if debug_level > 0:
+        print(text)
+
+
+def get_obs_from_ert(obs_data_from_ert_config):
+    obs_node_names = []
+    for obs_node in obs_data_from_ert_config:
+        node_name = obs_node.key()
+        obs_node_names.append(node_name)
+    debug_print("\n")
+
+    return obs_node_names
+
+
+def get_param_node_name_list_from_ert(ert):
+    ens_config = ert.ensembleConfig()
+    keylist = ens_config.alloc_keylist()
+    model_param_node_names = []
+    for key in keylist:
+        node = ens_config.getNode(key)
+        variable_type = node.getVariableType()
+        if variable_type == EnkfVarType.PARAMETER:
+            model_param_node_names.append(key)
+    return model_param_node_names
+
+
+def get_result_node_name_list_from_ert(ert):
+    ens_config = ert.ensembleConfig()
+    keylist = ens_config.alloc_keylist()
+    result_node_names = []
+    for key in keylist:
+        node = ens_config.getNode(key)
+        variable_type = node.getVariableType()
+        if variable_type == EnkfVarType.DYNAMIC_RESULT:
+            result_node_names.append(key)
+    return result_node_names
+
+
+class LocalisationConfigJob(ErtScript):
```

Comment:
list_all_obs_and_model_param.py should not be a part of any release of localisation. Its purpose is for me to check functionality of libres and to print out which observations and parameters are defined for an ERT project to simplify specification of localisation config files. So it is no point spending time on this file, but I have it in git to have version control over it. (A polished version of it could of course also be of help for end users, but that is another thing to be discussed with Eivind S and Tudor.

Reply:
It is removed from this PR